internal/categories/manager.go

```diff
@@ -46,6 +46,33 @@ func (m *Manager) Get(w http.ResponseWriter, r *http.Request) {
 	writeJson(w, fromDBTypeArray(dbCategories))
 }
 
+func (m *Manager) GetCategoryCounts(w http.ResponseWriter, _ *http.Request) {
+	var dtos map[string]CategoryCountDTO
+	dtos = make(map[string]CategoryCountDTO)
```

Comment:
You can declare this in one line with `dtos := make(map[string]CategoryCountDTO)`

Reply:
Ah yeah cool, will do